sdk/storage/azure-storage-blob-v2/src/main/java/com/azure/v2/storage/blob/BlobClient.java

```diff
@@ -0,0 +1,482 @@
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    BlobClient(BlobClientImpl serviceClient) {
+        this.serviceClient = serviceClient;
+    }
+
+    /**
+     * Sets properties for a storage account's Blob service endpoint, including properties for Storage Analytics and
+     * CORS (Cross-Origin Resource Sharing) rules.
+     * 
+     * @param options Options for setProperties API.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws StorageErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> setPropertiesWithResponse(SetPropertiesOptions options, RequestOptions requestOptions) {
+        return this.serviceClient.setPropertiesWithResponse(options, requestOptions);
+    }
+
+    /**
+     * Sets properties for a storage account's Blob service endpoint, including properties for Storage Analytics and
+     * CORS (Cross-Origin Resource Sharing) rules.
+     * 
+     * @param options Options for setProperties API.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws StorageErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void setProperties(SetPropertiesOptions options) {
+        this.serviceClient.setProperties(options);
+    }
+
+    /**
+     * Retrieves properties of a storage account's Blob service, including properties for Storage Analytics and CORS
+     * (Cross-Origin Resource Sharing) rules.
+     * 
+     * @param timeout The timeout parameter is expressed in seconds. For more information, see &lt;a
+     * href="https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/setting-timeouts-for-blob-service-operations"&gt;Setting
+     * Timeouts for Blob Service Operations.&lt;/a&gt;.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws StorageErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the service properties.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<StorageServiceProperties> getPropertiesWithResponse(Integer timeout,
```

Comment:
Should discuss with the service and SDK team on whether we want to expose the service side `timeout` here. I don't believe this is exposed at all in the current Track 2 libraries, and maybe there was good reason for that?